### DISCUSSION TITLE
How do I create a sponsorship profile?

### DISCUSSION THREAD

**Discussion by justdeen** [community/community#142681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I've been searching for how to create a sponsorship profile but I've not gotten any reasonable solution so I'll really appreciate if I get some feedback here.

**Comment by thedynomo** [comment]
1. Log in to your GitHub account.
2. Go to your profile.
3. Click "Sponsorships" (or search for "GitHub Sponsors").
4. Click "Get started".
5. Fill out your profile:
    - Bio (briefly describe yourself)
    - Profile picture
    - Goals (what you'll use funds for)
6. Set your sponsorship tiers (amounts and rewards)
7. Add a bank account or PayPal to receive funds.
8. Review and submit!

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/142683#discussioncomment-11078623) for reference to why this discussion was closed and locked. Thanks!
